virtualizarr/readers/dmrpp.py

```diff
@@ -82,24 +81,24 @@ class DMRParser:
         "String": "object",
     }
     # Default zlib compression value
-    _default_zlib_value = 6
+    _DEFAULT_ZLIB_VALUE = 6
     # Encoding keys that should be removed from attributes and placed in xarray encoding dict
-    _encoding_keys = {"_FillValue", "missing_value", "scale_factor", "add_offset"}
+    _ENCODING_KEYS = {"_FillValue", "missing_value", "scale_factor", "add_offset"}
```

Comment:
I just noticed this exact set also occurs here

https://github.com/zarr-developers/VirtualiZarr/blob/4ae7a19c2bb2996dc4739dfd3ebbe32b17ac1658/virtualizarr/writers/icechunk.py#L148

Perhaps we should make this a semi-private global variable defined in a common location? Perhaps move it to `virtualizarr/backend.py`?

Reply:
raised https://github.com/zarr-developers/VirtualiZarr/issues/290 to track this